### ISSUE TITLE
Filters say "out of date" or "new version available", but won't update

### ISSUE THREAD

**Issue by VLM1** [gorhill/uBlock#1895]
Greetings! I am using Pale Moon 26.3.3, win 7. I just installed uBlock Origin v 1.8.4 from the Firefox repository at https://addons.mozilla.org/en-US/firefox/addon/ublock-origin/. 

I went to the Dashboard, under 3rd-party filters, and added a few filters. I now have uBlock default set +  Fanboy’s Enhanced Tracking List‎ +  Fanboy’s Annoyance List‎ +  Fanboy+Easylist-Merged Ultimate List‎.

Then I noticed that some of the filters, including the "native" uBlock filters", were labeled either "out of date" or "new version available". So I clicked on "Update now", and all the labels changed to "Purge". So I clicked on "Purge all caches", and all the labels changed back. So I updated again, and all the labels changed to "Purge". 

In other words, I appear to be caught in some loop. Can anyone help?

Interestingly (perhaps), Fanboy's merged list does not show either out of date or purge.

Thanks!
[URL(s) for issue on a specific site are **mandatory**]

### Screenshot in which the issue can be seen

[Screenshot(s) for visual issues are **mandatory**]

### Steps for anyone to reproduce the issue

1. Be as detailed as possible:
1. Because we are not looking over your shoulder and
1. Keep in mind: Nobody can read your mind.

### Your settings

[If you fail to provide this info, I will mark the issue as invalid. Lists all settings which differs from default settings]

- Browser/version: 
- uBlock Origin version: 

##### Your filter lists

[Example: "Default filter lists + FRA", or "Default filter lists minus uBlock -- Badware risks"]

**Comment by gwarser** [comment]
You just removes your filters and then downloads them again - no issue here.

**Comment by gorhill** [comment]
When a list has the _purge_ button aside, this is not an instruction to click it. It's just that you can force a list to be updated if you removed its cache version.

**Comment by gorhill** [comment]
<https://github.com/gorhill/uBlock/wiki/Dashboard:-3rd-party-filters#update-now>

**Comment by VLM1** [comment]
Thank you both for responding. But I still have the core problem, which is that the lists won't update. ???

**Comment by gorhill** [comment]
You said:

> all the labels changed to "Purge"

Then this means all the lists were updated.

**Comment by VLM1** [comment]
Yes, I realize that's what it is supposed to mean. But as I wrote in  
the beginning of this thread, when I click "Purge" or "Purge all", the  
result returned still says "out of date", as before. Is this just a UI  
glitch, or are the filters actually not updating properly?

Thanks!

Quoting Raymond Hill <notifications@github.com>:

**Comment by gwarser** [comment]
Stop clicking on "Purge".

>Purge all caches

>This will remove all locally cached copies of filter lists. Essentially, this will cause all filter lists to become out of date.

**Comment by VLM1** [comment]
Okay, I did that before, and again now. But the labels still say "out of
date" on the same lists. Could there be a cookie-related problem here? Or
some other permissions problem?

Thank you for your continued participation in this conversation. I know
that you are busy, and I appreciate it!

BTW, are you the author of the Self-destructing Cookies extension? If so,
may I just add my 2 cents worth to say that this is a phenomenal product,
and that I hope you will consider continuing it for Pale Moon (and
whatever other Firefox forks continue to support XUL and the like), even
after Mozilla officially deprecates those tools. You have a fiercely loyal
and grateful band of followers in the PM community, and the thought of
losing SDC has engendered lengthy discussion, which I imagine has alerted
many more users to it.

FWIW, the only element of this extension that I wish were different is
this: I wish it were possible to set a combination of "allow 1st party
cookies only" and "keep for session". I would find that a useful policy
for many sites. Again just my 2 cents. :)

Kind regards,
Elchanan, for Vibrant Living Ministries


On Sun, August 14, 2016 9:17 pm, gwarser wrote:
Stop clicking on "Purge".

**Comment by VLM1** [comment]
Sorry, you must have thought I was out to lunch when you read my post,
below. The feature to which I made reference (explicit 1st-party cookies)
is in another extension called Cookie Controller, which I also use. I
really like the combination of explicit 1st-party cookies and removal on
tab closure.

Kind regards,

On Mon, August 15, 2016 12:59 pm, VLM TechForums wrote:
Okay, I did that before, and again now. But the labels still say "out of
date" on the same lists. Could there be a cookie-related problem here? Or
some other permissions problem?

Thank you for your continued participation in this conversation. I know
that you are busy, and I appreciate it!

BTW, are you the author of the Self-destructing Cookies extension? If so,
may I just add my 2 cents worth to say that this is a phenomenal product,
and that I hope you will consider continuing it for Pale Moon (and
whatever other Firefox forks continue to support XUL and the like), even
after Mozilla officially deprecates those tools. You have a fiercely loyal
and grateful band of followers in the PM community, and the thought of
losing SDC has engendered lengthy discussion, which I imagine has alerted
many more users to it.

FWIW, the only element of this extension that I wish were different is
this: I wish it were possible to set a combination of "allow 1st party
cookies only" and "keep for session". I would find that a useful policy
for many sites. Again just my 2 cents. :)

Kind regards,
Elchanan, for Vibrant Living Ministries


On Sun, August 14, 2016 9:17 pm, gwarser wrote:
Stop clicking on "Purge".
